gradle.properties

```diff
@@ -1,5 +1,6 @@
 kotlin.code.style=official
 xcodeproj=./iosHackthonApp
 android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
I'm curious, what does this do? 🤔 These are the bits of Droid dev I still really need to understand

Reply:
While Roger and I were hacking about with location we were getting some import errors / compatibility issues.  A search on google suggested adding Jetifier.  [Droid docs](https://developer.android.com/studio/command-line/jetifier)

Though I may try removing it and seeing whether it is still needed